lib/cylc/option_parsers.py

```diff
@@ -49,7 +50,7 @@ class CylcOptionParser(OptionParser):
 
 The old 'MATCH POINT' syntax will be automatically detected and supported. To
 avoid this, use the '--no-multitask-compat' option, or use the new syntax
-(with a '/' or a '.') when specifying 2 TASKID arguments."""
+(with a '/' or a '.') when specifying 2 TASK_GLOB arguments."""
```

Comment:
I wonder if we can get rid of compat for the old syntax?

Reply:
Would make life easier.

Reply:
Yes, get rid of it.  Cylc-8 is already disruptive, let's clear out old cruft.